tests/test_helpers.py

```diff
@@ -140,3 +144,17 @@ def test_get_elapsed_time(elapsed, expected):
 )
 def test_parse_strategy_name(name, expected):
     assert parse_strategy_name(name) == expected, f"Expected parsed strategy to be: {expected}."
+
+
+def test_create_folder_if_needed():
```

Comment:
@ZENALC Started adding some tests too but didn't wanna go too overkill yet.

![image](https://user-images.githubusercontent.com/713525/125187850-e3ea7380-e231-11eb-96ae-ed71df339d25.png)

Pycharms is reporting this metric